### PR TITLE
Delete Baidu_CN.xml

### PR DESCRIPTION
**Pull request by ivysrono**
#9906

### CODE REVIEW

File: src/chrome/content/rules/bdstatic.com.xml [hunk 1]
```diff
@@ -1,20 +1,17 @@
 <!--
 	For other Baidu coverage, see Baidu.xml.
 
-	Nonfunctional hosts in *bdstatic.com:
+	405 Not Allowed:
+		ns0.bdstatic.com	http://ns0.bdstatic.com/static/fisp_static/common/lib/mod_b818356.js
+		ns1.bdstatic.com	http://ns1.bdstatic.com/static/fisp_static/focustop/focustop/focustop_a273719.css
+
+	MCB:
+		ns2.bsdtatic.com
```
Comment (J0WI) [inline src/chrome/content/rules/bdstatic.com.xml]: typo: `bsdtatic` => `bdstatic`

File: src/chrome/content/rules/bdstatic.com.xml [hunk 2]
```diff
@@ -1,20 +1,17 @@
 <!--
 	For other Baidu coverage, see Baidu.xml.
 
-	Nonfunctional hosts in *bdstatic.com:
+	405 Not Allowed:
+		ns0.bdstatic.com	http://ns0.bdstatic.com/static/fisp_static/common/lib/mod_b818356.js
+		ns1.bdstatic.com	http://ns1.bdstatic.com/static/fisp_static/focustop/focustop/focustop_a273719.css
```
Comment (J0WI) [inline src/chrome/content/rules/bdstatic.com.xml]: Both urls WFM

Comment (ivysrono) [reply src/chrome/content/rules/bdstatic.com.xml]: I have updated this comment.
Same with you now?

Comment (J0WI) [reply src/chrome/content/rules/bdstatic.com.xml]: Nope, I get 200 on http and https

Comment (ivysrono) [reply src/chrome/content/rules/bdstatic.com.xml]: Test by Tor, Ok.
Directly, 405.
It's one China only issue.

**Pull request merged**